.github/workflows/tests.yml

```diff
@@ -377,6 +377,15 @@ jobs:
             # Silence lint error about backtick usage inside single quotes.
             # shellcheck disable=SC2016
             printf '::error ::`tests` job %s.\n' "$result"
+
+            deps_result='${{ needs.test_deps.result }}'
+            if [[ "$deps_result" = "skipped" ]]
+            then
+              # Silence lint error about backtick usage inside single quotes.
+              # shellcheck disable=SC2016
+              printf '::error ::`test_deps` job skipped. Do not merge until re-run with `CI-no-fail-fast-deps`\n'
+            fi
```

Comment:
I think this will also fail CI for formulae with no deps.

Reply:
It should only trigger when `tests` job failed or cancelled so CI would have already failed. Also doesn't impact the actual error code of step as it would `exit 1` whether or not this condition is hit.

Only thing is an extra unrelated message when no deps.